src/sidebar/components/group-list.js

```diff
@@ -23,9 +28,17 @@ function publisherProvidedIcon(settings) {
   return svc && svc.icon ? svc.icon : null;
 }
 
+/**
+ * @typedef GroupListProps
+ * @prop {(url: string) => any} [serviceUrl]
```

Comment:
I'd suggest to use the `ServiceUrlGetter` type from `sidebar/services/service-url.js` here.

Reply:
Yep, keep forgetting that 1